packages/gasket-cli/src/commands/create.js

```diff
@@ -169,7 +169,21 @@ comma-separated values: --preset-path=path1,path2`,
 Instead, prefer environment variables to configure package managers
 `,
     hidden: true
-  })
+  }),
+  'config': flags.string({
+    env: 'GASKET_PLUGINS',
+    description: `If provided, do not execute prompt lifecycle`,
```

Comment:
Match up to readme description:
```suggestion
    description: `JSON object that provides the values for any interactive prompts`,
```